tests/gflownet/envs/common.py

```diff
@@ -76,34 +120,94 @@ def test__get_parents_step_get_mask__are_compatible(env):
             assert mask[env.action_space.index(p_a)] is False
 
 
+@pytest.mark.repeat(500)
```

Comment:
This doesn't really do anything in this context.

Using this decorator will repeat the test 500 times but only when executed directly by pytest. When running these common test through `test__all_env_common()`, for instance, that decorator is ignored so the tests only run once each.

And since, to my knowledge, they are only ever invoked through `test__all_env_common()` or the continuous equivalent, that means that these decorators don't do anything.

Reply:
I wasn't aware of this. I think it is important to repeat some of these tests. Do you have any suggestions about how to better organise these common tests? Also, not a big deal, but because they are all part of `test__all_env_common()` they are only counted once.

Reply:
I will make this an issue so that it does not lock the merge, since it is not related to the PR but more general.

Reply:
Let's keep the conversation on the issue thread please: https://github.com/alexhernandezgarcia/gflownet/issues/242